twitter.html

```diff
@@ -0,0 +1,312 @@
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="twitter.css" />
+    <link rel="stylesheet" href="main-page.css" />
+    <link
+      rel="stylesheet"
+      href="news-feed.css
+    "
+    />
+  </head>
+  <body>
+    <div class="container">
+      <div class="task-bar">
+        <a class="twitter-logo" href="https://twitter.com/home">
+          <img
+            class="twitter-logo"
+            src="images\logo-twitter.svg"
+            alt="twitter
+          logo"
+          />
+        </a>
+
+        <a href="https://twitter.com/home"
+          ><img class="twitter-logo1" src="images\home.svg" alt="home icon"
+        /></a>
+        <a href="https://twitter.com/explore"
+          ><img
+            class="twitter-logo2"
```

Comment:
It might have been a good idea to name these something other than `twitter-logo2` so you could refer to them easier later.